mgmt/rest/tribe_test.go

```diff
@@ -206,6 +208,7 @@ func TestTribeTaskAgreements(t *testing.T) {
 						So(resp.Meta.Code, ShouldEqual, 200)
 						So(len(resp.Body.(*rbody.PluginList).LoadedPlugins), ShouldEqual, 1)
 						pluginToUnload := resp.Body.(*rbody.PluginList).LoadedPlugins[0]
+						<-lpe.pluginAddEvent
```

Comment:
This causes `wg.Wait()` to be called somewhere? Sorry, not trying to cause a ruckus just couldn't figure out where the test thread blocks waiting for all of the `wg.Done()`'s to roll in.